bin/stress-test/src/metrics.rs

```diff
@@ -0,0 +1,176 @@
+use std::{
+    fmt::Display,
+    path::{Path, PathBuf},
+    process::Command,
+    time::Duration,
+};
+
+const SQLITE_TABLES: [&str; 11] = [
+    "account_deltas",
+    "block_headers",
+    "account_fungible_asset_deltas",
+    "notes",
+    "account_non_fungible_asset_updates",
+    "nullifiers",
+    "account_storage_map_updates",
+    "settings",
+    "account_storage_slot_updates",
+    "transactions",
+    "accounts",
+];
+
+/// Metrics struct to show the results of the stress test
+pub struct Metrics {
+    insertion_time_per_block: Vec<Duration>,
+    get_block_inputs_time_per_block: Vec<Duration>,
+    get_batch_inputs_time_per_block: Vec<Duration>,
+    bytes_per_block: Vec<usize>,
+    num_insertions: u32,
+    store_file_sizes: Vec<u64>,
+    initial_store_size: u64,
+    store_file: PathBuf,
+}
+
+impl Metrics {
+    /// Create a new Metrics instance.
+    pub fn new(store_file: PathBuf) -> Self {
```

Comment:
nit: we usually use 3-rd person verbs in doc comments - e.g., `Create` -> `Creates`, `Track` - > `Tracks` etc.